ff/src/fields/arithmetic.rs

```diff
@@ -22,7 +23,7 @@ macro_rules! impl_field_mul_assign {
                 #[cfg(use_asm)]
                 #[allow(unsafe_code, unused_mut)]
                 {
-                    if $limbs <= 6 {
+                    if $limbs <= 6 && $limbs > 1 {
```

Comment:
Not related to this PR, but could there be a comment for why theres a different method for limbs > 6? Is it just that the larger limb size case isn't implemented? (Doesn't need to block this PR)

Reply:
Yes, larger limb size is not implemented